### PR TITLE
Ability for admin to creates athletes

### PR DESCRIPTION
**Pull request by jyuw**
#### Description
Changes proposed in this pull request:

* Ability to create users for the admin

#### What I have learned working on this feature: 
* Good recap how to work with cucumber again :)

### CODE REVIEW

File: features/admin_can_create_athlete.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    When I click "Create Athlete" button
+    Then I should be on the "Create Athlete" page
+    When I fill in "Name" with "Holger"
+    And I fill in "Home" with "Göteborg"
+    And I fill in "Age" with "26"
+    And I fill in "Starttime" with "1337"
+    And I attach a file
+    And I click "Create Athlete" button
+    And show me the page
+    Then I should see "Athlete successfully created"
```
Comment (holgertidemand) [inline features/admin_can_create_athlete.feature]: I think we should test the sad path aswell! :)

File: features/admin_can_create_athlete.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  I would like to be able to create athletes
+
+  Scenario: Admin successfully creates new athlete
+    Given I am on the "Dashboard" page
+    When I click "Create Athlete" button
+    Then I should be on the "Create Athlete" page
+    When I fill in "Name" with "Holger"
+    And I fill in "Home" with "Göteborg"
+    And I fill in "Age" with "26"
+    And I fill in "Starttime" with "1337"
```
Comment (holgertidemand) [inline features/admin_can_create_athlete.feature:13]: Let's try to fix this with an id . 
So we can write ``` Start Time```

**Review by holgertidemand — CHANGES_REQUESTED** [review]

**Review by holgertidemand — APPROVED** [review]

**Review by davidboll — APPROVED** [review]
LC&GTM

**Review by larathordar — COMMENTED** [review]
Please provide description to the pull request with a link to the pivitol tracker

**Pull request merged**